src/components/ui4/ButtonUi4.tsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * IMPORTANT: Changes in this file MUST be synced with edge-react-gui!
```

Comment:
We are already in edge-react-gui. Do you mean edge-login-ui-rn?

Reply:
I should just mention both to avoid copy pasta mistakes.